docker/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM centos:7
+MAINTAINER sqre-admin
+LABEL version="0.2.1" description="LSST DM/SQuaRE GitHub snapshot tools" \
+      name="lsstsqre/github-snapshot"
+
+USER root
+RUN yum update -y && \
```

Comment:
It's less that it's a big build time win and more that I still remember some absurdly low limit on number of layers.  That's probably been fixed since a year or more ago when it mattered, particularly since it's unlikely this is going to be a base for future work.

Still...I like collapsing consecutive RUN statements into one big one.